app/views/chat/_users.html.erb

```diff
@@ -8,7 +8,11 @@
     </tr>
     </thead>
     <tbody>
-    <% counter = 1%>
+    <% if params[:page].to_i === 0%>
+      <% counter = 1 %>
+    <% else %>
+      <% counter = 1 + (5 * (params[:page].to_i - 1)) %>
```

Comment:
use kaminari for pagination (users)
add https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-cycle zebra-style to highlight rows like that https://www.google.com/search?q=stripped+table&client=ubuntu&hs=o39&channel=fs&sxsrf=ALiCzsYP5os5lq4t41TWn1gmudTaxl6KeQ:1658309611500&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjvtcjalIf5AhUOx4sKHSX6APQQ_AUoAXoECAEQAw&biw=1969&bih=1258&dpr=1.5#imgrc=8p8XZY6sQO8quM